### ISSUE TITLE
Launching ARCore SceneViewer from a native application

### ISSUE THREAD

**Issue by eugenelozada** [google-ar/arcore-android-sdk#786]
I posted a feature request #760 for a built in functionality in ARCore that allowed for a built in AR viewer to preview AR content.  This was delivered in 1.9 with the release of SceneViewer and the [documentation](https://developers.google.com/ar/develop/java/scene-viewer) for that showed how to enable web sites to invoke it.  Now I didn't get a response regarding how to enable this for native apps, so I did a little digging in the logs when trying out the [demo site](http://model-viewer.glitch.me/) and found

` I/ActivityManager: START u0 {act=android.intent.action.VIEW cat=[android.intent.category.BROWSABLE] dat=https://cdn.glitch.com/... flg=0x14000000 pkg=com.google.ar.core cmp=com.google.ar.core/.viewer.ViewerActivity (has extras)} from uid 10111`

Which seemed relatively straightforward so I wrote a simple app that just fired a matching intent:

```
val uri = Uri.parse("https://cdn.glitch.com/36cb8393-65c6-408d-a538-055ada20431b/Astronaut.glb?1542147958948")
intent = (Intent.ACTION_VIEW, uri)
```

And true enough, it allowed me to choose ARCore to open the .glb file and when chosen, launched SceneViewer.  So my question is - this going to be an 'officially' supported feature and if so, can we have more documentation around it?

**Comment by dsternfeld7** [comment]
The intent is undocumented because it is currently subject to change at any time. You are welcome to use it. However, do so at your own risk of a breaking change. There is a more detailed discussion of this [here](https://github.com/GoogleWebComponents/model-viewer/issues/547)

**Comment by eugenelozada** [comment]
Thank you for the response.  I understand that this is a work in progress, but can I make an assumption that eventually this will be made public?

Reason is I need to make a decision whether or not to continue building our own 'standard', reusable AR viewer that we can launch from web or a native app.

**Comment by satputerohan** [comment]
@eugenelozada, 
I can see arcore content on the web just fine with the device. So it's definitely installed. I checked my packages on adb. The exception is thrown for not having an activity that can start the intent when trying from my app.
There are two activities that are fired in the logs on web. As soon as you click the scene viewer button, the .viewer.IntentForwardActivity is fired, which I tried, but same error is thrown. 
Did you add something to your manifest file to get the intent to fire? I am not a native Android developer, so my knowledge is limited. I work with Xamarin Forms.
